gradle/plugins/src/main/kotlin/org/cqfn/diktat/buildutils/diktat-convention-configuration.gradle.kts

```diff
@@ -1,37 +0,0 @@
-package org.cqfn.diktat.buildutils
-
-import org.cqfn.diktat.plugin.gradle.DiktatJavaExecTaskBase
-
-plugins {
-    id("org.cqfn.diktat.diktat-gradle-plugin")
-}
-
-diktat {
-    diktatConfigFile = rootProject.file("diktat-analysis.yml")
-    githubActions = findProperty("diktat.githubActions")?.toString()?.toBoolean() ?: false
-    inputs {
-        // using `Project#path` here, because it must be unique in gradle's project hierarchy
-        if (path == rootProject.path) {
-            include("gradle/plugins/src/**/*.kt", "*.kts", "gradle/plugins/**/*.kts")
-            exclude("gradle/plugins/build/**")
-        } else {
-            include("src/**/*.kt", "**/*.kts")
-            exclude(
-                "src/test/**/*.kt",
-                "src/test/**/*.kts",
-                "src/*Test/**/*.kt",
-                "build/**/*.kts",
-            )
-        }
-    }
-}
-
-tasks.withType<DiktatJavaExecTaskBase>().configureEach {
```

Comment:
this configuration is not required in new plugin

Reply:
yep